h/accounts/test/views_test.py

```diff
@@ -32,6 +32,8 @@ def configure(config):
     config.registry.registerUtility(SubmitForm, IProfileForm)
     config.registry.registerUtility(MagicMock(), IRegisterSchema)
     config.registry.registerUtility(MagicMock(), IRegisterForm)
+    config.registry.feature = MagicMock()
```

Comment:
Ok that line is actually crashing on master currently (but there's not test failing).

@tilgovi @nickstenning I'm not sure how Pyramid controllers are supposed to access `feature()` now? Back when we were doing the original atom stream feature, there was an `includeme()` in `api_client/__init__.py` that would add a request method to the request object for getting an api client. includeme() would access api_url from settings to do this. But the idea was that view code didn't need to access settings, it just used the api client that had been placed on the request object for it.

So what about features? I could have an includeme call feature() and add a notifications_feature boolean to the request but this seems silly as we just removed feature() from request and we're gonna end up polluting the request object with a lot of new feature on/off properties.

What's the intention here? (Can't find an example in the code)